### ISSUE TITLE
Padding parameter without fill filter is not working

### ISSUE THREAD

**Issue by EleonoreD** [cshum/imagor#314]
Hello, 

I'm trying to use the `padding` parameter (`GxH:IxJ`) to add some empty spaces around my picture but it does not work if i don't also add a `fill:(color)` filter. 

But I want this space to be `transparent` and filling with `"transparent"` does not seems to work either.

For example : 

```
https://[endpoint]/0x0:347x253/1400x1050/150x50:150x50/filters:orient(360)/[...filePath...]
// -> does not apply the padding `150x50:150x50`

https://[endpoint]/0x0:347x253/1400x1050/150x50:150x50/filters:fill(orange):orient(360)/[...filePath...]
// -> apply the padding `150x50:150x50` but with an orange background

https://[endpoint]/0x0:347x253/1400x1050/150x50:150x50/filters:fill(transparent):orient(360)/[...filePath...]
// -> apply the padding `150x50:150x50` but with a black background, `transparent` not recognize ?

```

Do you have any solution for me or is it possible to make `"transparent"` filling available ? 

Thanks for your answers !

**Comment by cshum** [comment]
You may test the docker build at master branch:
https://github.com/cshum/imagor/pkgs/container/imagor/69987041?tag=master

```
ghcr.io/cshum/imagor@sha256:9b9b1a7d54d5a6721187ffe91eb37369b1de3dd02f1c02dc3244978f2c3918e6
```

**Comment by cshum** [comment]
available in v1.4.0

**Comment by EleonoreD** [comment]
Thanks for this upgrade ! It works well ! 👍
